### PR TITLE
Make dasdfmt more robust

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: tests/installation/bootloader_s390.pm [hunk 1]
```diff
@@ -164,16 +164,14 @@ sub format_dasd() {
     select_console('install-shell');
 
     # bring dasd online
-    script_run("dasd_configure 0.0.0150 1");
+    assert_script_run("dasd_configure 0.0.0150 1");
 
     # make sure that there is a dasda device
-    assert_script_run("(lsdasd | tee /dev/$serialdev)");
+    assert_script_run("lsdasd \| tee /dev/$serialdev");
```
Comment (dzedro) [inline tests/installation/bootloader_s390.pm]: hmm backslash ? Is so windowsish

Comment (Soulofdestiny) [inline tests/installation/bootloader_s390.pm]: good point,  will check

**Comment by Soulofdestiny** [comment]
@dzedro tested and updated

**Comment by okurz** [comment]
saw the verification run

**Pull request merged**